gprofiler/profiler_state.py

```diff
@@ -40,3 +44,17 @@ def insert_dso_name(self) -> bool:
     @property
     def profiling_mode(self) -> str:
         return self._profiling_mode
+
+    @property
+    def container_names_client(self) -> Optional[ContainerNamesClient]:
+        return self._container_names_client
+
+    def refresh_container_cache(self) -> None:
+        if self._container_names_client:
```

Comment:
```suggestion
        if self._container_names_client is not None:
```

Please fix everywhere - no implicit `None`/bool conversions